.gitignore

```diff
@@ -33,3 +33,5 @@ coverage
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
```

Comment:
Actually, maybe we want master.key to be checked in, now?  I guess it comes down to how "secret" are any development or test passwords?  I would guess there aren't any I care about people getting hold of.  But maybe you can think of something I'm not?

Reply:
I think there may be dev secrets that we don't want checked in, especially if i resume work on the GraphQL api, because in dev mode we will conceivably need to test giving viable credentials to some API or another. 

I'm also wary of automated code scanners, especially after that automated alert this morning, that i was "exposing" smtp credentials. I don't think we want bots scouring the repo and finding credentially-looking things or master keys.

`master.key` is automatically added to .gitignore by Rails when you first set up credentials. My motive for starting to use them is just to keep moving our setup closer to a Rails default, so potential devs don't need to spend time deciphering the basics of it.

Reply:
Also we should probably be running `git secret` in general and it would get grumpy about that.  Doesn't seem like a big deal to share around the master.key outside of git as needed.